bridges/scalajs-1.0/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -114,24 +122,19 @@ object JsBridge {
       env: Map[String, String]
   ): (List[sbt.testing.Framework], () => Unit) = {
     implicit val debugFilter: DebugFilter = DebugFilter.Test
-    val config = NodeJSConfig().withExecutable(nodePath).withCwd(Some(baseDirectory)).withEnv(env)
-    val nodeEnv =
-      if (!jsdom) new NodeJSEnv(logger, config)
-      else new JsDomNodeJsEnv(logger, config)
-
-    val outputJsContents = {
-      val contents = java.nio.file.Files.readAllBytes(jsPath)
-      MemVirtualBinaryFile(jsPath.toString, contents)
-    }
-
-    // The order of the scripts mandates the load order in the js runtime
-    val inputs = List(outputJsContents)
     val nodeModules = baseDirectory.resolve("node_modules").toString
     logger.debug("Node.js module path: " + nodeModules)
     val fullEnv = Map("NODE_PATH" -> nodeModules) ++ env
+    val config =
+      NodeJSConfig().withExecutable(nodePath).withCwd(Some(baseDirectory)).withEnv(fullEnv)
+    val nodeEnv =
+      if (!jsdom) new NodeJSEnv(logger, config)
+      else new JsDomNodeJsEnv(logger, config)
 
+    // The order of the scripts mandates the load order in the JavaScript runtime
+    val input = Seq(Input.Script(jsPath))
```

Comment:
Unfortunately, we cannot access the module kind in this function. Since it is for discovering testing frameworks, hard-coding `Input.Script` should be fine.

Reply:
You should add a parameter for the `moduleKind` if you cannot otherwise access it. It's really necessary. Even discovering the testing frameworks requires to be able to *load* the file, and that will not work if you use a CommonJS module or an ES module as a script.

Reply:
Thanks for clarifying. I fixed it.